### DISCUSSION TITLE
The data source of ET_Ref and  EW_ref

### DISCUSSION THREAD

**Discussion by ylbestwishes** [iiasa/CWatM#88] [Q&A] [answered]
Hello! Sincerely thanks for your reading! I am wondering what is the data source of ET_REF(Potential reference evapotranspiration rate) and EW_ref(Potential evaporation rate from water surface). I want to substitute the data of rhine in CWatM-Earth-30min folder with my basin file. I would appreciate it so much for your reply!

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, these are created as a CWatM outputs when running using all of the climate variables. 
One can run CWatM using all the climate variables or using ETRef and EWRef when they have been created. Running with ETRef and EWRef is somewhat faster.

We discuss creating these as part of this [video](https://youtu.be/aEmPx-KPFZ4).

First, to use all the climate variables, use the climate data linked in the CWatM-Earth-30min repository description. The settings file in this repository is set up already to use these climate data. In this case, the settings 
**calc_evaporation = True**

Then, run for your basin using the coordinates of the outlet or a mask map. We demonstrate using output coordinates with these climate data in this [video](https://youtu.be/vlpKfV_w8LE).

Then include ETRef and EWRef as daily output maps:
**OUT_MAP_Daily = ETRef, EWRef**

When these have been created, in the settings file 
**calc_evaporation = False**
**E0Maps = _path to_ EWRef.nc**
 **ETMaps = _path to_ ETRef.nc**

**Reply by ylbestwishes** [reply]
Thanks so much! I have solved my problem. Thanks again!
